lightning/src/ln/onchaintx.rs

```diff
@@ -771,11 +817,47 @@ impl<ChanSigner: ChannelKeys> OnchainTxHandler<ChanSigner> {
 		if let Some(ref local_commitment) = self.local_commitment {
 			if local_commitment.has_local_sig() { return Err(()) }
 		}
+		if self.local_htlc_sigs.is_some() || self.prev_local_htlc_sigs.is_some() {
```

Comment:
Isn't that what it implies with "local comitment transaction has been signed"? I'm a bit confused by your comment here.

Reply:
Nevermind, I think it was to make the distinction between a "local commitment has been signed" and a "local commitment has been _by-this-monitor_". But if you see a local commitment onchain, even broadcast from another monitor, the "local" copy should have been signed & broadcast because trigger condition should be met too (aka block height).

We should care about "local" (not remote commitment) and "local"(aka on this watchtower).